api/models/templates/app.tmpl

```diff
@@ -332,7 +332,11 @@
         "SecurityGroupIngress": [
           {{ range .PortMappings }}
             {
+              {{ if .Public }}
               "CidrIp": "0.0.0.0/0",
+              {{ else }}
+              "CidrIp": "10.0.0.0/16",
```

Comment:
I see two strategies here.

1. Use a go template variable and pass `os.Getenv("VPCCIDR")` into the go template rendering context.
2. Add VPCCIDR as an app stack cloudformation parameter, and use a cloudformation `{ "Ref": "VPCCIDR" }'